script.js

```diff
@@ -15,13 +17,224 @@ navPort.addEventListener('click', () => {
 });
 
 navAbout.addEventListener('click', () => {
-    navbar.classList.toggle('active');
-    fixed.classList.toggle('fixed-active');
+  navbar.classList.toggle('active');
+  fixed.classList.toggle('fixed-active');
+});
+
+navContact.addEventListener('click', () => {
+  navbar.classList.toggle('active');
+  fixed.classList.toggle('fixed-active');
+});
+
+const CardArray = [{
+  title: 'Multi-Post Stories Gain+Glory',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/detail-snapshot.png',
+},
+
+{
+  title: ' Keeping track of components',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/modal-desk-pic.png',
+},
```

Comment:
- Please on your desktop view, the popup elements are not properly aligned as in the Figma design. PLease try to fix the below issues.
  - The right side margin between the `X` button and the project container is too small compared to the Figma design, try increasing the right margin to march that of the Figma design.
  - Please the popup image is centered and has a smaller width compared to that of the Figma design, please try to fix it so it looks like that of the Figma design.
  - Please proper align the text in the popup so that it has similar padding on both sides as the Figma design. 

##### Your design
![Screenshot from 2022-06-16 18-33-20](https://user-images.githubusercontent.com/58771507/174132995-0fc218c4-6ee0-4ae5-bb9a-a176a33ee913.png)

##### Figma design 
![Screenshot from 2022-06-16 18-33-39](https://user-images.githubusercontent.com/58771507/174133055-0e515bb8-fb77-4561-8b0d-a7ce0dab39f9.png)

Reply:
Thank you for pointing those out, I will check it out.